app/src/main/java/com/x8bit/bitwarden/data/platform/manager/FeatureFlagManagerImpl.kt

```diff
@@ -20,10 +20,7 @@ class FeatureFlagManagerImpl(
     override val sdkFeatureFlags: Map<String, Boolean>
         get() = mapOf(
             CIPHER_KEY_ENCRYPTION_KEY to
-                isServerVersionAtLeast(
-                    serverConfigRepository.serverConfigStateFlow.value,
-                    CIPHER_KEY_ENC_MIN_SERVER_VERSION,
-                ),
+                getCipherKeyEncryptionFlagState(),
```

Comment:
If the flag changes, this will not be applied until the app restarts. I assume we are OK with that.

Reply:
Yes it is acceptable in this case.